client/card-readers/settings/file-upload.js

```diff
@@ -0,0 +1,136 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import wcpayTracks from 'tracks';
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '@wordpress/i18n';
+import { useDispatch } from '@wordpress/data';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { FileUploadControl } from 'wcpay/disputes/evidence/file-upload';
```

Comment:
How about extracting `FileUploadControl` out of the "disputes" folder and making it available at a common location?

Reply:
Is this solved @mordeth ?

Reply:
As there may be changes to file upload, that one is left for the next phase. Will be added to the follow-up ticket for phase 2.

Reply:
:+1: